torch/ao/quantization/backend_config/fbgemm.py

```diff
@@ -71,6 +72,49 @@
 )
 
 
+# ===================
+# | FBGEMM OP CONFIGS |
+# ===================
+
+
+def _get_fbgemm_op_configs(dtype_configs: List[DTypeConfig]) -> List[BackendPatternConfig]:
+    configs = []
+    fbgemm_ops = [
+        torch.nn.InstanceNorm1d,
+        torch.nn.InstanceNorm2d,
+        torch.nn.InstanceNorm3d,
+        torch.nn.LayerNorm,
```

Comment:
If we're adding functional `group_norm` anyway, should we also add `torch.nn.GroupNorm` here? @jerryzh168

Reply:
yeah I think we can do that since we have quantized module for these things: https://github.com/pytorch/pytorch/blob/master/torch/ao/nn/quantized/modules/normalization.py, maybe better to do this in a separate PR.

Reply:
I can make a PR to add it in later.